### DISCUSSION TITLE
Email notification when a part is changed

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#7819] [Q&A]
I see it's possible to subsribe to a part, but whether it's possible to receive an email when a part is changed?

**Comment by SchrodingersGat** [comment]
Not by default, no. The plugin system has access to these events, and could be used to send out an email, but currently there is no implementation of this in the core code base.

If this is something you want to see, perhaps you can contribute this feature?

**Comment by jesseyu1984918** [comment]
Yes, I can help contribute to this but need some guidance, or I can fund this.On Aug 7, 2024, at 6:25 PM, Oliver ***@***.***> wrote:﻿
Not by default, no. The plugin system has access to these events, and could be used to send out an email, but currently there is no implementation of this in the core code base.
If this is something you want to see, perhaps you can contribute this feature?

**Reply by SchrodingersGat** [reply]
Ok I have created a new issue to address this - https://github.com/inventree/InvenTree/issues/7834
